jsapp/scss/components/_kobo.connect-projects.scss

```diff
@@ -0,0 +1,208 @@
+.connect-projects {
```

Comment:
Please move this file to `jsapp/js/components/<whatever you choose to be feature directory name>` :)

Reply:
Also I don't think we need the `_kobo.` prefix in the names anymore :coffin: